src/action.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
I don't think we should do this as we won't be running these files as executables. Is this needed for an editor or something?

Reply:
Fixed. Left #!/usr/bin/env python3 only for main.py.